src/v1/driver.js

```diff
@@ -72,6 +73,21 @@ class Driver {
      * @protected
      */
     this._connectionProvider = null;
+
+    this._onCompleted = null;
+  }
+
+  get onCompleted() {
+    return this._onCompleted;
+  }
+
+  set onCompleted(callback) {
```

Comment:
Setting an event handler invoking actual verification confused me a bit. What happens if verification fails? Maybe an explicit method `verifyConnectivity()` returning a promise would be a better option, can be elaborated for upcoming versions?

Reply:
Failed verification will result in `Driver.onCompleted` not being invoked. Instead `Driver.onError` will be invoked. I agree, these callbacks are not ideal and rather ugly. That's why they are removed from README. Users can verify connectivity by running a dummy query like `RETURN 1`. So this fix is only for backwards compatibility.